dsi/run-from-dsi

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+pushd "$(dirname "$0")/../../genny/" >/dev/null
+    GENNY_REPO_ROOT="$(pwd -P)"
+    export GENNY_REPO_ROOT
+popd >/dev/null
+
+pushd "$(dirname "$0")/.." >/dev/null
+    WORKLOAD_REPO_ROOT="$(pwd -P)"
+    export WORKLOAD_REPO_ROOT
+popd >/dev/null
+
+"${GENNY_REPO_ROOT}/run-genny" "auto-tasks" "--tasks" "${@}" "--workload-repo" "${WORKLOAD_REPO_ROOT}"
```

Comment:
1. I'm confused about why we need `workload-repo`
2. Similarly, Genny knows the genny repo root by convention. It's always `./src/genny`.
3. What args are being passed in? Not sure I'm following that `--tasks` takes a list of args (`${@}`) versus just the specific task type that's being generated.